taxi/urls.py

```diff
@@ -42,10 +47,33 @@
     path("cars/create/", CarCreateView.as_view(), name="car-create"),
     path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
+    path(
+        "cars/<int:pk>/add_cur_driver/",
+        add_cur_driver_to_car,
+        name="car-add-cur-driver"
+    ),
+    path(
+        "cars/<int:pk>/del_cur_driver/",
+        del_cur_driver_to_car,
+        name="car-del-cur-driver"
+    ),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path(
+        "drivers/create/", DriverCreateView.as_view(), name="driver-create"
+    ),
+    path(
+        "drivers/<int:pk>/update/",
+        DriverUpdateView.as_view(),
+        name="driver-update"
+    ),
+    path(
+        "drivers/<int:pk>/delete/",
+        DriverDeleteView.as_view(),
+        name="driver-delete"
+    ),
```

Comment:
Could you please follow the style format code? The current formatting is a bit messy